src/Mod/Path/PathScripts/PathAreaOp.py

```diff
@@ -43,8 +44,8 @@
 __doc__ = "Base class and properties for Path.Area based operations."
 __contributors__ = "russ4262 (Russell Johnson)"
 __createdDate__ = "2017"
-__scriptVersion__ = "2p"
-__lastModified__ = "2020-02-13 17:11 CST"
+__scriptVersion__ = "3a"
+__lastModified__ = "2020-02-16 15:11 CST"

```

Comment:
Hi Russ, its not good practice holding version information and dates in the code. No other developers use this so we can end up with changes to the code being made without the version changing or other developers have to remove it when they make changes to the code.

Reply:
Done.

Thanks.

Reply:
I agree with dubstar. We don't need to track the version in the source file because we are using Git already. Git is precisely a version control system, it uses hashes to identify modifications to the source code, and we can always go back to the version of a particular hash. Tracking the changes manually can be done but it's work that the developer doesn't really need to do, Git does it for us.